tests/unit/test_organisation.py

```diff
@@ -9,13 +9,42 @@
 
 from __future__ import unicode_literals
 
+import pytest
 from mock import patch
 from tornado.testing import AsyncTestCase, gen_test
 
 from perch import User, Organisation, Service
+from voluptuous import MultipleInvalid
 from .util import make_future
 
 
+TEST_REFERENCE_LINKS = {
```

Comment:
Maybe also add a test to make sure not having a redirect_id_type still validates